packages/loader/container-loader/src/container.ts

```diff
@@ -2432,10 +2442,39 @@ export class Container
 		return { snapshot, versionId: version?.id };
 	}
 
+	private async fetchSnapshot(
+		specifiedVersion: string | undefined,
+	): Promise<{ snapshot?: ISnapshot | ISnapshotTree; versionId?: string }> {
+		if (
+			this.mc.config.getBoolean("Fluid.Container.FetchSnapshotInNewFormat") === true &&
+			this.service?.policies?.supportNewSnapshotFormat === true
```

Comment:
do we need supportNewSnapshotFormat? It feels like presence of getSnapshot API should be enough for that, no?
Maybe there are some cases where we have API but it's NYI (i.e. throws). If possible, it would be great not to have that (and basically have one truth on whether it's supported or not)

Reply:
I have added comment on the prop in interface, but I just want to have it and we will enable it in the driver once the driver fully supports the new api as there could be time when we have this api, but internally it is not fully implemented. Also, we can just leave a hook here in case someone wants to disbale it from app side, by passing this policy just in case. We can always remove it later.